backend/python/coqui/coqui_server.py

```diff
@@ -66,7 +66,19 @@ def LoadModel(self, request, context):
 
     def TTS(self, request, context):
         try:
-            self.tts.tts_to_file(text=request.text, speaker_wav=self.AudioPath, language=COQUI_LANGUAGE, file_path=request.dst)
+            # if model is multilangual add language from request or env as fallback
+            lang = request.Lang or COQUI_LANGUAGE
```

Comment:
Can I add a new `Lang` field in the protobuf definition ? It would be an optional one.

Reply:
If the language is truly independent of both the model, voice, and input text, I see no reason **not** to have a `Language` parameter. Personally, I prefer to spell it out rather than name it `Lang`?

Reply:
Agreed better to have clearly defined parameter. 

Does it make sense to keep the  COQUI_LANGUAGE env var ? What use case does it serve ?